src/autoevals/llm.py

```diff
@@ -181,7 +181,7 @@ def __new__(cls, model=None, use_cot=None, max_tokens=None, temperature=None):
     return C
 
 
-for model in MODEL_TEMPLATES:
-    globals()[model] = _build_template_class(model)
+for model_template in MODEL_TEMPLATES:
```

Comment:
@ankrgyl - `model` is a param name in some of the classes above causing conflict. Renamed this variable to avoid conflict.

Reply:
Ah an even safer thing to do would be:

```
def _init_model_templates():
  for model_template in MODEL_TEMPLATES:
     ...

_init_model_templates()
```

(python is much better about function scope than block / loop scope)

Reply:
Updated: https://github.com/braintrustdata/autoevals/pull/3/commits/ede368311c534574c3ff8d2e4f4099429a03608a#diff-514aef479a03d7c505ddbd0300d768802d78018c5312a94063d061f3bccba3a2R184